opencog/moses/main/populate_atomspace.h

```diff
@@ -41,6 +41,14 @@ using namespace combo;
  */
 void populate(AtomSpace *as,  ITable &itable);
 
+/**
+ * Populate the given atomspace with input features of compressed table.
+ * @param Atomspace&     as
```

Comment:
I'm no doxygen expert (or even user) but I was under the impression that the format was
```
@param <argument-name> <description>
```
but again, I'm no doxygen user so...